lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -101,6 +101,18 @@ static bool peelType(Type type, SmallVectorImpl<FlatBundleFieldEntry> &fields) {
       .Default([](auto op) { return false; });
 }
 
+/// Return if something is a subaccess, special-cased for subaccess taking
+/// constants, which are really subindex.
+static bool isNotSubAccess(Operation *op) {
```

Comment:
I think the comment should be "if something is _not_ a subaccess"?

Reply:
I guess it's more complicated than that. Maybe just: "Return true if an operation is not a subaccess or should not be lowered as a subaccess because it is a subaccess of a literal expression."